packages/expo-sqlite/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### 🐛 Bug fixes
 
 - Fixed exceptions when converting empty blob data on iOS. ([#33564](https://github.com/expo/expo/pull/33564) by [@kudo](https://github.com/kudo))
+- Fixed kv-store promises not well transfered when using AsyncStorage compatible api ([#33847](https://github.com/expo/expo/pull/33847) by [@rtorrente](https://github.com/rtorrente))
```

Comment:
```suggestion
- Fixed `expo-sqlite/kv-store` async API not being well handled and having database locked error occasionally ([#33847](https://github.com/expo/expo/pull/33847) by [@rtorrente](https://github.com/rtorrente))
```

if the pr is trying to resolve the database locked error. maybe we could just write it down.